runtime/dolphin/src/lib.rs

```diff
@@ -707,11 +728,14 @@ construct_runtime!(
         TechnicalMembership: pallet_membership::<Instance2>::{Pallet, Call, Storage, Event<T>, Config<T>} = 18,
 
         // Collator support. the order of these 5 are important and shall not change.
+        // Session must be included *before* AuthorInherent
+        AuthorInherent: pallet_author_inherent::{Pallet, Call, Storage, Inherent} = 60,
+        AuraAuthorFilter: pallet_aura_style_filter::{Pallet, Storage} = 63,
```

Comment:
Do we really need this pallet if it's just going to call back into `CollatorSelection` ?
`CollatorSelection` -> `AuraFilter` -> `CollatorSelection`

Reply:
Gotcha, thanks.